content/ru/details.md

```diff
@@ -58,6 +58,7 @@ description: "Hacktoberfest Ninja - Checker - Hacktoberfest представле
 - Плохие репозитории будут исключены. В прошлом, мы замечали репозитории которые вдохновляли участников создавать достаточно лёгкие pull requestы. К примеру добавляли их имена в файл чтобы быстро получить pull request который засчитывался в челендж Hacktoberfestа. Хотя это может быть инструментом обучения для новых участников, это идет вразрез с одной из наших основных ценностей Hacktoberfest. Качество pull requestов первостепенно; колличество идёт потом. Подобные репозитории не мотивируют усердных вложений и предоставляют нечестное преимущество в завершении Hacktoberfest испытания. Мы применили систему блокирующую подобные репозитории и любой pull request отправленные в них не будет засчитываться.
 
 [Более подробную информацию о том, как жаловаться на репозитории который не следует правилам Hacktoberfestа можно найти на официальном сайте.](https://hacktoberfest.digitalocean.com/details#spam)
+[Английская версия этого файла details.md](https://github.com/lukeocodes/hacktoberfest-checker/blob/main/content/en/details.md)
```

Comment:
please dont add additional content because it might not reflect the other translations